holoviews/tests/util/test_transform.py

```diff
@@ -520,3 +525,56 @@ def test_pickle(self):
         expr = (((dim('float')-2)*3)**2)
         expr2 = pickle.loads(pickle.dumps(expr))
         self.assertEqual(expr, expr2)
+
+
+@pytest.fixture    
+def df_fixture():
+    """Prepare dataframe fixture.
+
+    Returns:
+        pd.DataFrame: Output data with 
+    """
+    
+    data_text = (
+        """
+        idx| name|   a|    b
+        101|   P1|  7.|   3.
+        102|   P2|  3.|   4.
+        103|   P3|  .5|   3.
+        104|   P4|  2.|   2.
+        105|   P5|  1.|   2.
+        106|   P6|  1.|   1.
+        """)
+
+    # read in data using non-zero based index
+    return pd.read_csv(StringIO(data_text), sep='|', skipinitialspace=True, index_col=0)
+ 
+    
+
+def test_dataset_transform_by_spatial_select_expr_index_not_0_based(df_fixture):
```

Comment:
The df_fixture should be removed, and create the DataFrame inside this test.

Reply:
Ok, then I will remove the fixture.